content/enterprise_influxdb/v1.8/guides/hardware_sizing.md

```diff
@@ -0,0 +1,342 @@
+
+For OSS hardware sizing guidelines, click on following [LINK](https://docs.influxdata.com/influxdb/v1.8/guides/hardware_sizing/).
+
+> **Disclaimer:** Your numbers may vary from recommended guidelines. Guidelines provide estimated benchmarks for implementing the most performant system for your business.
+
+## Enterprise overview
+
+InfluxDB Enterpise supports the following:
+
+- more than 750,000 field writes per second
+- more than 100 moderate queries per second ([see Query guides](#query-guidelines))
+- more than 10,000,000 [series cardinality](/influxdb/v1.8/concepts/glossary/#series-cardinality)
+
+InfluxDB Enterprise distributes multiple copies of your data across a cluster,
+providing high-availability and redundancy, so an unavailable node doesn’t significantly impact the cluster.
+Please [contact us](https://www.influxdata.com/contact-sales/) for assistance tuning your system.
+
+If you want a single node instance of InfluxDB that's fully open source, requires fewer writes, queries, and unique series than listed above, and do **not require** redundancy, we recommend InfluxDB OSS.
+
+> **Note:** Without the redundancy of a cluster, writes and queries fail immediately when a server is unavailable.
+
+## Query guidelines
+
+> Query complexity varies widely on system impact. Recommendations for both single nodes and clusters are based on **moderate** query loads.
+
+For **simple** or **complex** queries, we recommend testing and adjusting the suggested requirements as needed. Query complexity is defined by the following criteria:
+
+| Query complexity | Criteria                                                                              |
+|------------------|---------------------------------------------------------------------------------------|
+| Simple           | Have few or no functions and no regular expressions                                   |
+|                  | Are bounded in time to a few minutes, hours, or 24 hours at most                      |
+|                  | Typically execute in a few milliseconds to a few dozen milliseconds                   |
+| Moderate         | Have multiple functions and one or two regular expressions                            |
+|                  | May also have `GROUP BY` clauses or sample a time range of multiple weeks             |
+|                  | Typically execute in a few hundred or a few thousand milliseconds                     |
+| Complex          | Have multiple aggregation or transformation functions or multiple regular expressions |
+|                  | May sample a very large time range of months or years                                 |
+|                  | Typically take multiple seconds to execute                                            |
+
+## InfluxDB Enterprise cluster guidelines
+
+### Meta nodes
+
+> Set up clusters with an odd number of meta nodes──an even number may cause issues in certain configurations.
```

Comment:
```suggestion
> Set up clusters with an odd number of meta nodes─an even number may cause issues in certain configurations.
```

Reply:
let's check all three hardware sizing docs for the extra punctuation in line 62 (should be one em-dash here)